### PR TITLE
Skip inverse_norm in moments

### PR DESCRIPTION
**Pull request by Ukyeon**
1. Since inverse_norm is only required for the layer X, it is recommended to comment out the function in the moments section. This is because the moments function is designed to run specifically on layers starting with 'X_'.
2. Fix for debug message issue in cell_velocities.

### CODE REVIEW

**Comment by Xiaojieqiu** [comment]
@Ukyeon so now we don't perform any transform on the X_spliced and X_unspliced layers right? That is, no log1p will be used by default?

I think `adata.uns["pp"]["norm_method"]` should actually keep the normalization method used for different layers which should be consistent for all layers.

File: dynamo/tools/utils.py [hunk 1]
```diff
@@ -991,28 +991,31 @@ def inverse_norm(adata: AnnData, layer_x: Union[np.ndarray, sp.csr_matrix]) -> n
         The inverse normalized data.
     """
 
+    if layer_x == "X":
```
Comment (Sichao25) [inline dynamo/tools/utils.py:994]: We may need layer X inverse_norm, we can use something like `norm_method_key = "X_norm_method" if layer is None or layer == "X" else "layers_norm_method"`.

File: dynamo/tools/metric_velocity.py [hunk 2]
```diff
@@ -67,13 +67,13 @@ def cell_wise_confidence(
...
+            adata.X if X_data is None else X_data,  ##### Check! inverse_norm for adata.X?
             adata.layers[vkey] if V_data is None else V_data,
         )
-        norm_method = adata.uns["pp"]["norm_method"].copy()
-        adata.uns["pp"]["norm_method"] = "log1p"
+        norm_method = adata.uns["pp"]["layers_norm_method"].copy()
+        adata.uns["pp"]["layers_norm_method"] = "log1p"
         X = inverse_norm(adata, X) if X_data is None else X_data
-        adata.uns["pp"]["norm_method"] = norm_method
+        adata.uns["pp"]["layers_norm_method"] = norm_method
```
Comment (Sichao25) [inline dynamo/tools/metric_velocity.py:76]: It seems that we are performing inverse_transform on X here.

File: dynamo/tools/utils.py (hunk 1 above)
Comment (Ukyeon) [reply dynamo/tools/utils.py:994]: @Xiaojieqiu 
Can you please confirm that we still need to do inverse_norm for adata.X ?

**Pull request merged**